pkg/dispatchcli/cmd/manage.go

```diff
@@ -0,0 +1,164 @@
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"path"
+
+	"github.com/spf13/cobra"
+	"github.com/vmware/dispatch/pkg/dispatchcli/i18n"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/mitchellh/go-homedir"
+	restclient "k8s.io/client-go/rest"
+)
+
+var (
+	manageShort = `Manage Dispatch configurations.`
+	manageLong  = `Manage Dispatch configurations.`
+
+	manageExample = i18n.T(`
+# Enable Dispatch bootstrap mode while specifying bootstrap user and associated public key
+dispatch manage --enable-bootstrap-mode --bootstrap-user admin@example.com --public-key ./app.rsa.pub
+# Disable Dispatch bootstrap mode
+dispatch manage --disable-bootstrap-mode`)
+
+	enableBootstrapModeFlag  = false
+	disableBootstrapModeFlag = false
+
+	bootstrapUser = ""
+
+	kubeconfigPath = ""
+)
+
+const (
+	bootstrapSecretName = "dispatch-identity-manager-bootstrap"
+)
+
+// NewCmdManage creates a command object for Dispatch "manage" action
+func NewCmdManage(out, errOut io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     i18n.T(`manage [--enable-bootstrap-mode | --disable-bootstrap-mode]`),
+		Short:   manageLong,
+		Long:    manageLong,
+		Example: manageExample,
+		Run: func(cmd *cobra.Command, args []string) {
+			err := runManage(out, errOut, cmd, args)
+			CheckErr(err)
+		},
+	}
+	cmd.Flags().BoolVarP(&enableBootstrapModeFlag, "enable-bootstrap-mode", "e", false, "enable Dispatch bootstrap mode")
+	cmd.Flags().BoolVarP(&disableBootstrapModeFlag, "disable-bootstrap-mode", "d", false, "disable Dispatch bootstrap mode")
+	cmd.Flags().StringVar(&bootstrapUser, "bootstrap-user", "", "specify bootstrap user")
+	cmd.Flags().StringVar(&publicKeyPath, "public-key", "", "public key file path for bootstrap user")
+	cmd.Flags().StringVar(&kubeconfigPath, "kubeconfig", "", "customized absolute path to k8s config file (optional)")
+	return cmd
+}
+
+// prepare k8s client throuhg kubeconfig flag or default kubeconfig location
+func prepareK8sClient() (clientset *kubernetes.Clientset, err error) {
+
+	var configPath string
+	var config *restclient.Config
+
+	// create k8s config
+	if kubeconfigPath != "" {
+		configPath, err = homedir.Expand(kubeconfigPath)
+	} else {
+		homeDir, _ := homedir.Dir()
+		configPath = path.Join(homeDir, ".kube", "config")
+	}
+	config, err = clientcmd.BuildConfigFromFlags("", configPath)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to load k8s config: %s", err.Error())
+	}
+
+	// create k8s clientset
+	clientset, err = kubernetes.NewForConfig(config)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to create k8s client: %s", err.Error())
+	}
+	return clientset, nil
+}
+
+func updateBootstrapSecret(secret *v1.Secret, client *kubernetes.Clientset) error {
+	var err error
+	existed, err := client.CoreV1().Secrets("dispatch").Get(bootstrapSecretName, metav1.GetOptions{})
```

Comment:
We can't assume `dispatch` is going to be the namespace.
Options:-
1. Ask the user the install config file and read the dispatch namespace
2. or ask for the namespace where dispatch is installed.

I think it's ok to ask for (1)

Reply:
fixed, ask for install config and get namespace from it